packages/next-polyfill-nomodule/src/index.js

```diff
@@ -20,7 +37,10 @@ import 'core-js/features/number/is-nan'
 import 'core-js/features/number/is-safe-integer'
 import 'core-js/features/number/max-safe-integer'
 import 'core-js/features/number/min-safe-integer'
+import 'core-js/features/number/parse-float'
+import 'core-js/features/number/parse-int'
 import 'core-js/features/object/entries'
+import 'core-js/features/object/from-entries'
```

Comment:
This one seems to have only landed support in major browsers after the nomodule cutoff point, meaning we can't polyfill it in this way
https://caniuse.com/#feat=mdn-javascript_builtins_object_fromentries
https://caniuse.com/#feat=mdn-api_htmlscriptelement_nomodule

Reply:
That's interesting, thank you! Is there a method in place to load polyfills for both modern and legacy browsers?

Reply:
I've removed this from the nomodule polyfills and set the TypeScript library definitions' version to ES2018.

Reply:
I tried my best to [ship a tiny polyfill for every browser](https://github.com/vercel/next.js/pull/15772/commits/dc4a2abc2deb0397c46fce1aee1f052252d681ec), similarly to how `Promise.prototype.finally` gets loaded.

Reply:
I don't think we should increase bundle sizes for all browsers for this feature given it's likely barely used in user code. It should be polyfilled on-demand

Reply:
Okay, I've removed it.